Cargo.toml

```diff
@@ -47,6 +48,7 @@ pin-project-lite = "0.1.7"
 route-recognizer = "0.2.0"
 serde = "1.0.102"
 serde_json = "1.0.41"
+surf = { version = "2.0.0-alpha.5", default-features = false, optional = true, features = ["h1-client"] } # TODO(jbr): remove this feature when surf can build without default features
```

Comment:
Tying `tide` to a specific version of `surf` is likely not ideal. Other proposals have managed to avoid this, and I think if we can we should do so.